src/components/Admin/resources/competition/publication/PublicationCreate.tsx

```diff
@@ -0,0 +1,49 @@
+import {FC} from 'react'
+import {
+  AutocompleteInput,
+  FileInput,
+  NumberInput,
+  ReferenceInput,
+  required,
+  SelectInput,
+  SimpleForm,
+  TextInput,
+} from 'react-admin'
+import {useSearchParams} from 'react-router-dom'
+
+import {MyCreate} from '@/components/Admin/custom/MyCreate'
+import {MyFileField} from '@/components/Admin/custom/MyFileField'
+import {Accept} from '@/utils/dropzone-accept'
+
+import {createPublicationFormData} from './createPublicationFormData'
+
+export const PublicationCreate: FC = () => {
+  const [searchParams] = useSearchParams()
+
+  return (
+    <MyCreate
```

Comment:
1. MyCreate sa po vytvoreni snazi redirectnut na "show", ale ten neexistuje, preto to skonci na edite... ktory ma hore ShowButton, ktory teda tiez nefunguje.
2. asi by sme mali pridat moznost vratit sa na ten event. najjednoduchsie by asi bolo doimplementovat Show a tam mat reference link na Event